build-image/conf/sql-client-conf.yaml

```diff
@@ -148,6 +148,37 @@ tables:
     primary-key: taxiId
     time-attribute: usageStartTime
 
+  # Define a dimension table here, have the same schema with DriverChanges, but it will store the full-snapshot of drivers.
+  - name: Drivers2
+    type: temporal-table
+    update-mode: append
+    schema:
+      - name: taxiId
+        type: LONG
+      - name: driverId
+        type: LONG
+      - name: usageStartTime
```

Comment:
We don't need timestamp field, because it is not a changelog, but a lastest table.